website/package.json

```diff
@@ -6,21 +6,21 @@
   "scripts": {
     "clean": "gatsby clean",
     "check-links": "start-server-and-test 'yarn serve' http://localhost:9000/custom-applications '../.linkcheck/linkcheck.sh --skip-file ../.linkcheck/skip_list.txt http://localhost:9000/custom-applications'",
-    "build": "./scripts/prebuild.sh && yarn generate-icons && gatsby build --prefix-paths --verbose --log-pages && yarn check-links",
+    "build": "./scripts/prebuild.sh && yarn generate-icons && gatsby build --prefix-paths --verbose --log-pages",
```

Comment:
Is it intended to not check links at build time anymore?

Reply:
Yes we do check it against the preview URL

https://github.com/commercetools/merchant-center-application-kit/blob/182f8f4f7be4936f236161815fc810c4d2f255b8/.github/workflows/docs.yml#L124-L146

Testing links against the local build is not necessary on CI.